nautobot/extras/models/jobs.py

```diff
@@ -1127,6 +1131,17 @@ def schedule(self):
 
         return self.to_cron()
 
+    @property
+    def queue(self):
+        return self.job_queue.name if self.job_queue else ""
+
+    @queue.setter
+    def queue(self, value):
+        if value:
+            self.job_queue, _ = JobQueue.objects.get_or_create(
```

Comment:
Same question here about create-by-assignment, can/should we change this to just `JobQueue.objects.get(...)`?

Reply:
If queue does not exist, raise a validation error same as above.